robolectric/src/test/java/org/robolectric/shadows/ShadowCanvasTest.java

```diff
@@ -529,4 +530,36 @@ public void restoreToCount() {
 
     assertThat(canvas.getSaveCount()).isEqualTo(save1);
   }
+
+  @Test
+  public void drawRect_toCanvas() {
```

Comment:
What about using name `drawRect_withPureFloatPosition`?

Reply:
ok, i do it..